### PR TITLE
Retouch bullet printing

### PR DESCRIPTION
**Pull request by MadmanMartian**
## About The Pull Request

Makes ammo disks give ammo boxes/strips, rather than magazines

Magazines are now printed on a seperate disk

## Why It's Good For The Game

Substantially better bullet economy

Magazines are now worth something to players, and don't have misleading names

## Changelog
:cl:
add: Adds a generic magazine autolathe disk
tweak: Lethal and nonlethal disks no longer print magazines, instead they print boxes of ammunition
/:cl:

### CODE REVIEW

File: code/game/objects/items/weapons/design_disks/frozen_star.dm [hunk 1]
```diff
@@ -1,60 +1,67 @@
...
-		//smg mags
-		/datum/design/autolathe/ammo/smg/rubber,
-		//magnum smg mags
-		/datum/design/autolathe/ammo/msmg/rubber,
-		//rifles
-		/datum/design/autolathe/ammo/srifle/rubber,
-		/datum/design/autolathe/ammo/ihclrifle/rubber,
-		/datum/design/autolathe/ammo/lrifle/rubber,
+		/datum/design/autolathe/ammo/pistol_ammobox/rubber = 4,
+		/datum/design/autolathe/ammo/sl_magnum/rubber = 4,
```
Comment (MadmanMartian) [inline code/game/objects/items/weapons/design_disks/frozen_star.dm:10]: Accidentally added a speedloader rather than an ammo packet

**Comment by Voldirs** [comment]
But there are already ammo box disks, pistol and rifle ammo ones. Also it's possible to print full mags with gun disks

**Comment by MadmanMartian** [comment]
Yes.

**Review by genessee596 — CHANGES_REQUESTED** [review]
No. Let's not add more complexity for the sake of complexity. It's already a pain in the ass to fumble around with half a dozen disks - instead of making players sift through even more disks, add that 'ammo vendor' we talked about that fills up inserted magazines for a credit cost.

**Review by Firefox13 — CHANGES_REQUESTED** [review]
Even if Genessee isn't a delegate anymore I support his opinion - there should be a way to reload a magazine for a fee from a vendor.

**Comment by Firefox13** [comment]
Closing this PR due to inactivity. Feel free to re-open it if you want to address the requested changes.

**Pull request closed without merging**